test/diff/diff_files/spec_constant_specid_dst.spvasm

```diff
@@ -0,0 +1,13 @@
+               OpCapability Shader
+               OpMemoryModel Logical GLSL450
+               OpEntryPoint GLCompute %main "main"
+               OpExecutionMode %main LocalSize 1 1 1
+       %void = OpTypeVoid
+          %3 = OpTypeFunction %void
+       %uint = OpTypeInt 32 0
+     %v3uint = OpTypeVector %uint 3
+         %ss = OpSpecConstant %uint 10
+       %main = OpFunction %void None %3
+          %5 = OpLabel
+               OpReturn
+               OpFunctionEnd
```

Comment:
Use a real editor Spencer ;)

(no new line at end of file)

Reply:
Fixed, also added `OpSpecConstantOp` support as well